src/components/cylc/Mutation.vue

```diff
@@ -18,55 +18,57 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <template>
     <v-card class="d-inline-block pa-4">
       <!-- the mutation title -->
-      <h3 :style="{ 'text-transform': 'capitalize' }">
-        {{ mutation._title }}
-      </h3>
-
-      <!-- the mutation description -->
-      <v-expansion-panels
-        variant="accordian"
-        v-bind="extendedDescription ? { hover: true } : { readonly: true }"
-      >
-        <v-expansion-panel
-          class="mutation-desc"
-          elevation="0"
+      <v-card-title>
+        <h3 :style="{ 'text-transform': 'capitalize' }">
+          {{ mutation._title }}
+        </h3>
+      </v-card-title>
+      <v-card-text class="card-text">
+        <!-- the mutation description -->
+        <v-expansion-panels
+          variant="accordian"
+          v-bind="extendedDescription ? { hover: true } : { readonly: true }"
         >
-          <v-expansion-panel-title
-            v-bind="extendedDescription ? {} : {
-              expandIcon: null,
-              style: {
-                cursor: 'default'
-              }
-            }"
+          <v-expansion-panel
+            class="mutation-desc"
+            elevation="0"
           >
-            <Markdown :markdown="shortDescription"/>
-          </v-expansion-panel-title>
-          <v-expansion-panel-text v-if="extendedDescription">
-            <Markdown :markdown="extendedDescription"/>
-          </v-expansion-panel-text>
-        </v-expansion-panel>
-      </v-expansion-panels>
-
-      <v-divider />
-      <EditRuntimeForm
-        v-if="mutation.name === 'editRuntime'"
-        v-bind="{
-          cylcObject,
-          types
-        }"
-        ref="form"
-        v-model="isValid"
-      />
-      <FormGenerator
-        v-else
-        v-bind="{
-          mutation,
-          types,
-          initialData
-        }"
-        ref="form"
-        v-model="isValid"
-      />
+            <v-expansion-panel-title
+              v-bind="extendedDescription ? {} : {
+                expandIcon: null,
+                style: {
+                  cursor: 'default'
+                }
+              }"
+            >
+              <Markdown :markdown="shortDescription"/>
+            </v-expansion-panel-title>
+            <v-expansion-panel-text v-if="extendedDescription">
+              <Markdown :markdown="extendedDescription"/>
+            </v-expansion-panel-text>
+          </v-expansion-panel>
+        </v-expansion-panels>
+        <v-divider />
+        <EditRuntimeForm
+          v-if="mutation.name === 'editRuntime'"
+          v-bind="{
+            cylcObject,
+            types
+          }"
+          ref="form"
+          v-model="isValid"
+        />
+        <FormGenerator
+          v-else
+          v-bind="{
+            mutation,
+            types,
+            initialData
+          }"
+          ref="form"
+          v-model="isValid"
+        />
+      </v-card-text>
       <br />
       <v-card-actions>
```

Comment:
While you're here I've noticed this break element is not needed. And we can add back in some padding I've removed from the parent card above.

```suggestion
      <v-card-actions class="pa-3">
```